image/nginx/nginx.conf

```diff
@@ -0,0 +1,100 @@
+user www-data;
+worker_processes 4;
+pid /run/nginx.pid;
```

Comment:
`pid` location is different in previous version, but like the `user` field, it works with this path.